controllers/openshift_controller.go

```diff
@@ -102,10 +104,125 @@ func (r *KataConfigOpenShiftReconciler) Reconcile(ctx context.Context, req ctrl.
 			return ctrl.Result{}, updateErr
 		}
 
+		ds := r.processDaemonsetForMonitor()
+		// Set KataConfig instance as the owner and controller
+		if ds != nil {
+			r.Log.Info("daemonset is not nil")
```

Comment:
"daemonset exists", maybe even pass a bit of useful data to the log?

("not nil" is internal jargon)

Reply:
Another related point that @fgiudici asked in the original PR: can r.processDaemonsetForMonitor() actually return "nil", and in what condition ? I guess it can, as Jens made sure to requeue if it does end up being nil (which I think means we will just retry later). But I'm not sure in what conditions.
So in addition to more data in the "success" case here, we probably also want to get more info on the "fail" part below, in case somehow we end up in a retry loop.

Reply:
I have tried to address that, and the other "info" strings in the same area.
Unfortunately, I am not sure what additional information I could put in there. Any suggestion is welcome.